<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecipeBook</class>
 <widget class="QMainWindow" name="RecipeBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Книга рецептов</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="searchBar">
      <property name="placeholderText">
       <string>Поиск...</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="searchByNameRB">
        <property name="text">
         <string>По названию</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="searchByIngredientRB">
        <property name="text">
         <string>По ингредиенту</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="recipeList">
      <property name="iconSize">
       <size>
        <width>150</width>
        <height>150</height>
       </size>
      </property>
      <property name="isWrapping" stdset="0">
       <bool>false</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="addRecipeButton">
      <property name="text">
       <string>Добавить рецепт</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>655</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Справка</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
